pallets/randomness/src/lib.rs

```diff
@@ -134,6 +134,9 @@ pub mod pallet {
 		/// Babe requests expire and can be purged from storage after this many blocks/epochs
 		#[pallet::constant]
 		type EpochExpirationDelay: Get<u64>;
+		/// Maximum number of random seeds stored for previous rounds.
+		#[pallet::constant]
+		type MaxPreviousLocalRandomness: Get<u8>;
```

Comment:
After talking with @4meta5 it's probable that we can remove this parameter and simply use the current high-entropy Local VRF as the returned seed.

Reply:
There may still need to be some delay cc @crystalin so maybe it is the local VRF output from N blocks ago instead of hashing together the last N outputs

The reason for this is that author-selection needs to be able to predict authoring eligibility.